jekyll/_docs/install-and-use-yarn.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: classic-docs
+title: "Install and Use Yarn (the NPM replacement) on CircleCI"
+short-title: "Install and Use Yarn on CircleCI"
+categories: [how-to]
+description: "How to modify `circle.yml` in order to install and use Yarn on CircleCI."
+---
+
+<img src="{{site.baseurl}}/assets/img/logos/yarn-logo.svg" style="display:block;margin:15px auto;width:40%;min-width:320px;" alt="Yarn Logo" />
+
+[Yarn][yarn-site] is an open-source package manager for JavaScript. CircleCI can cache both Yarn and the packages it installs. This can potentially speed up builds but, more importantly, can reduce errors related to network connectivity.
+
+## Setup
+
+```
+machine:
+  environment:
+    YARN_VERSION: 0.17.8
+    PATH: "${PATH}:${HOME}/.yarn/bin"
+```
+
+We've set the Yarn version here to reduce the number of moving parts in the build. Although the Yarn install script sets its `PATH`, when we're potentially running from cache, we have to explicitly add `~/.yarn/bin/`.
+
+## Installing Yarn
+
+```
+dependencies:
+  pre:
+    - |
+      if [[ ! -e ~/.yarn/bin/yarn || $(yarn --version) != "${YARN_VERSION}" ]]; then
+        curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version $YARN_VERSION
+      fi
+  cache_directories:
+    - ~/.yarn
+    - ~/.yarn-cache
+```
+
+Here, the Yarn install script is run if and only if:
+
+* Yarn isn't already installed.
+* The installed version of Yarn doesn't match the specified version in the machine section.
+
+`~/.yarn` is specified as a cached directory for Yarn itself. `~/.yarn-cache` is specific as a cached directory for Yarn packages.
```

Comment:
Because these two sentences are so similar, I would bold the differences:

"**`~/.yarn`** is specified as a cache directory for **Yarn itself**. **`~/.yarn-cache`** is specified as a cache directory for **Yarn packages**."